MOxUnit/@MOxUnitTestReport/getDuration.m

```diff
@@ -0,0 +1,18 @@
+function duration=getDuration(obj)
+% get the duration of the test
+%
+% duration=getDuration(obj)
+%
+% Input:
+%   obj                 MOxUnitTestReport instance
+%
+% Output:
+%   duration            total time in seconds that it took to run all tests
+%                       in the obj instance
+
+    duration=0;
+
+    for i=1:countTestOutcomes(obj)
+        test_outcome=getTestOutcome(obj,i);
+        duration=duration+getDuration(test_outcome);
+    end
```

Comment:
This could be generalised into an arbitrary function, like `cellfun` but for MOxUnitTestReport objects. Obviously only consider it if you're likely to loop over these objects in a couple of other places and not just here.